packages/block-library/src/file/save.js

```diff
@@ -11,11 +12,31 @@ export default function save( { attributes } ) {
 		textLinkTarget,
 		showDownloadButton,
 		downloadButtonText,
+		displayPreview,
+		previewHeight,
 	} = attributes;
 
 	return (
 		href && (
 			<div { ...useBlockProps.save() }>
+				{ displayPreview && (
+					<>
+						<object
+							className="wp-block-file__embed"
+							data={ href }
+							type="application/pdf"
+							style={ {
+								width: '100%',
+								height: `${ previewHeight }px`,
```

Comment:
Same goes here for previous files about `previewHeight`. If we only toggle `displayPreview` it ends up with `undefinedpx`